x-pack/legacy/plugins/ml/public/application/services/http_service.ts

```diff
@@ -4,68 +4,66 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-// service for interacting with the server
+import { Observable } from 'rxjs';
 
-import { fromFetch } from 'rxjs/fetch';
-import { from, Observable } from 'rxjs';
-import { switchMap } from 'rxjs/operators';
-
-import { getXSRF } from '../util/dependency_cache';
```

Comment:
`getXSRF` can now be removed from `dependency_cache` and all references to `XSRF` can go including the `__LEGACY` section from `MlDependencies`

Reply:
Check https://github.com/elastic/kibana/pull/59320/commits/bae457f24724df9118a4380484000df8ba40aa1d